Makefile

```diff
@@ -118,6 +118,12 @@ validate-test-list:
 	@echo "Comparing newly generated docs to existing docs"
 	cnf-tests/hack/compare-gen-md.sh
 
+verify-commits:
```

Comment:
This can go under the `ci-job` makefile rule

Reply:
ah, this is easier than adding an additional lane then, great, will do

Reply:
done